config/default.json

```diff
@@ -34,7 +34,7 @@
       "analytics": {
         "url": "https://data.getodk.cloud/v1/key/eOZ7S4bzyUW!g1PF6dIXsnSqktRuewzLTpmc6ipBtRq$LDfIMTUKswCexvE0UwJ9/projects/1/submission",
         "formId": "odk-analytics",
-        "version": "2022.11.29.01"
+        "version": "2023.01.22.01"
```

Comment:
I haven't created this form version yet, but I will! First I wanted to validate my thinking that the field should be removed.